swagger.yaml

```diff
@@ -4390,6 +4390,8 @@ definitions:
         type: string
         x-nullable: false
         default: 'x86_64'
+        # TODO: remove arm64 when AI moves to using aarch64
+        enum: ['','x86_64', 'aarch64', 'arm64','ppc64le','s390x','multi']
```

Comment:
What purpose does the empty value in this enum serve if there's also a default set?
(applies for all instances of this in the swagger file).

Reply:
Ack, missed that... Thanks.